configure.py

```diff
@@ -1638,14 +1604,11 @@ def make_bin_override():
 
 config = '\n'.join(['='.join(item) for item in config.items()]) + '\n'
 
-# On Windows there's no reason to search for a different python binary.
-bin_override = None if sys.platform == 'win32' else make_bin_override()
-if bin_override:
-  config = 'export PATH:=' + bin_override + ':$(PATH)\n' + config
+os.environ['PYTHON_EXECUTABLE'] = sys.executable
 
 write('config.mk', do_not_edit + config)
 
-gyp_args = ['--no-parallel', '-Dconfiguring_node=1']
+gyp_args = ['--no-parallel', '-Dconfiguring_node=1', "-DPYTHON_EXECUTABLE=" + os.path.realpath(sys.executable)]
```

Comment:
```suggestion
gyp_args = ['--no-parallel', '-Dconfiguring_node=1', "-DPYTHON_EXECUTABLE=%s" % sys.executable]
```
AFAIK `sys.executable` is already an absolute path, so it doesn't need further resolving.
(I prefer string formatting to `+`, since it will coerce the parameter to `str`)